.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Safety-CLI Dev Container",
+    "build": {
+        "dockerfile": "Dockerfile",
+        "context": "..",
+        "args": {
+            "SAFETY_VERSION": "DEV"
+        }
+    },
+    "settings": {
+        "terminal.integrated.shell.linux": "/bin/bash"
+    },
+    "extensions": [
+        "ms-python.python",
+        "ms-python.vscode-pylance",
+        "ms-python.debugpy"
+    ],
+    "postCreateCommand": "pip install -r test_requirements.txt && pip install ruff requests pre-commit",
```

Comment:
if we go with adding that non-root user, could consider using it here potentially:
```
"remoteUser": "appuser",
```